templates/cli/README.md.twig

```diff
@@ -43,21 +43,17 @@ $ {{ language.params.executableName }} init
 
 The CLI requires an endpoint, project ID, API key and a locale to be able to communicate with the {{ spec.title | caseUcfirst }} backend server. There are two ways to achieve this 
 
-### 1. Environment file 
-```sh
-$ {{ language.params.executableName }} --env-file path/to/your/environment/file init     
-```
-The .env file must have the following keys 
+### 1. Pass in the project settings along with `appwrite init`
+
 ```sh
-endpoint=http://192.168.1.6/v1
-X-{{ spec.title | caseUcfirst }}-Project=5ff450422d42f
-X-{{ spec.title | caseUcfirst }}-Key=0773c7...a2672a
-X-{{ spec.title | caseUcfirst }}-Locale=en-US
+$ {{ language.params.executableName }} init --endpoint="YOUR ENDPOINT" --project="YOUR PROJECT ID" --key="YOUR API KEY" --locale="YOUR LOCALE"
 ```
 
+If any of the required values are missing, a prompt will show up, asking for those missing values.
+
 ### 2. Using the prompt
 
-Invoking `{{ language.params.executableName }} init` without an env file triggers an interactive prompt where you can manually enter the details. These values are saved locally so that you dont have to re-enter them everytime. 
+Invoking `{{ language.params.executableName }} init` without an params triggers an interactive prompt where you can manually enter the details. These values are saved locally so that you dont have to re-enter them everytime. 
```

Comment:
`without an params triggers an interactive prompt where you can manually enter the details` fix grammar

Reply:
Good catch 😄